{"name": "test_api_petstore", "status": "failed", "statusDetails": {"message": "AssertionError: No coincide\nassert 'Juancho' == 'John'\n  - John\n  + Juancho", "trace": "self = <tests.test_pet_store_get.PetStore testMethod=test_api_petstore>\n\n    def test_api_petstore(self):\n        '''Este tests verifica, a través del método GET, que los datos que devuelve la petición a la api sean\n        correctos'''\n    \n        response = requests.get(self.url)\n        json_response = json.loads(response.text)\n        print(json.dumps(json_response, indent=3))\n    \n        assert response.status_code == 200\n        assert json_response['id'] == 2, 'No coincide'\n>       assert json_response['name'] == 'John', 'No coincide'\nE       AssertionError: No coincide\nE       assert 'Juancho' == 'John'\nE         - John\nE         + Juancho\n\ntests/test_pet_store_get.py:21: AssertionError"}, "description": "Este tests verifica, a través del método GET, que los datos que devuelve la petición a la api sean\n        correctos", "attachments": [{"name": "stdout", "source": "fe591c30-163f-41cc-9378-d582cf2e9cf4-attachment.txt", "type": "text/plain"}], "start": 1594840832742, "stop": 1594840833515, "uuid": "1131924e-2bd5-4313-aeee-be47247e6fdb", "historyId": "f2917ce1750e5b45370039c7b7733629", "testCaseId": "636c8908fde22675bdf03ef5b7695723", "fullName": "tests.test_pet_store_get.PetStore#test_api_petstore", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_pet_store_get"}, {"name": "subSuite", "value": "PetStore"}, {"name": "host", "value": "WKSAR128L.local"}, {"name": "thread", "value": "7750-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_pet_store_get"}]}